esignet-default.properties

```diff
@@ -389,8 +391,8 @@ mosip.esignet.ui.config.key-values={'sbi.env': 'Developer', 'sbi.timeout.DISC':
   'consent.screen.timeout-buffer-in-secs': 5, 'linked-transaction-expire-in-secs': 240, 'sbi.port.range': 4501-4600, \
   'sbi.bio.subtypes.iris': 'UNKNOWN', 'sbi.bio.subtypes.finger': 'UNKNOWN', 'wallet.qr-code-buffer-in-secs': 10, 'otp.length': 6, \
   'password.regex': '\\S*', 'wallet.config': ${mosip.esignet.ui.wallet.config},\
-  'auth.factor.kba.individual-id-field' : ${mosip.esignet.authenticator.default.auth-factor.kba.individual-id-field},\
-   'auth.factor.kba.field-details':${mosip.esignet.authenticator.default.auth-factor.kba.field-details} }
+  'auth.factor.kba.individual-id-field' : '${mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.individual-id-field}',\
+  'auth.factor.kba.field-details':${mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.field-details}}
```

Comment:
Undo this change, let it point to default configs

Reply:
done